docs/en/OLED_Display.md

```diff
@@ -0,0 +1,199 @@
+# Oled Display
+Extension to use for your build with OLED Display
+
+If you need help soldering, there's a small note on the end to help.
+
+# Preparation
+
+### Libraries
+First of all you need to download few Libraries that will make possible for your OLED to work. You can download them from Adafruit CircuitPython Libraries page. 
+#### Make sure you download right version of Libraries. It should be the same one as yours CircuitPyhon.
+* [CircuitPython Libraries Bundle](https://circuitpython.org/libraries)
+
+Now you need to create `lib` folder where you will place your needed files.
+Then find following folder and file and drop them in your freshly baked folder.
+* adafruit_display_text
+* `adafruit_displayio_ssd1306.mpy`
+
+# Main.py
+Time to make changes in `main.py`.
+As always, first step is adding OLED extension as well as busio and board.
+
+```python
+import board
+import busio
+from kmk.extensions.oled import Oled, TextEntry, ImageEntry
+```
+
+Now add this main part of extension. Then replace SCL and SDA with correct pins/
+
+```python
+i2c_bus = busio.I2C(board.GP SCL, board.GP SDA) # change SCL and SDA according to your board and made connection.
+```
+
+Here's the main part with all of the settings, you can tweak them to fit your screen and preferences.
+
+```python
+oled = Oled(
+    i2c=i2c_bus,
+    device_address=0x3C,
+    width=128, # Screen size
+    height=64, # Screen size
+    flip = False, # Flips your display content
+    flip_left = False, # Flips your display content on left side split
+    flip_right = False, # Flips your display content on right side split
+    dim_time=10, # time in seconds to reduce screen brightness
+    dim_target=0.1, # set level for brightness decrease
+    off_time=0, # time in seconds to turn off screen
+    brightness=1, # initial screen brightness level
+    brightness_step=0.1, # used for brightness increase/decrease keycodes
+    # POWER SAVE ONLY SETTINGS
+    powersave_dim_time=10, # time in seconds to reduce screen brightness
+    powersave_dim_target=0.1, # set level for brightness decrease
+    powersave_off_time=30, # time in seconds to turn off screen
+)
+```
```

Comment:
Merge these code examples.
Keep constistent upper/lowercase at the beginning of a comment.